### PR TITLE
Allow ?:

### PR DESCRIPTION
**Pull request by adrian-enspired**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,7 +44,7 @@
   - Never nest ternary statements
   - Wrap the condition in ()
   - Must be contained on either one or three lines
-  - Never use the shorthand version `?:`
+  - The shorthand `?:` may be used as a substitute for the expression `empty($a) ? $a : $b`
```
Comment (robertology) [inline README.md]: `! empty()`

but that isn't completely true since `$a` has to be defined

Comment (adrian-enspired) [reply README.md]: I didn't mean to imply that, but will add a note.

File: README.md [hunk 2]
```diff
@@ -44,7 +44,7 @@
   - Never nest ternary statements
   - Wrap the condition in ()
   - Must be contained on either one or three lines
-  - Never use the shorthand version `?:`
+  - The shorthand `?:` may be used as a substitute for the expression `empty($a) ? $a : $b` (note, `$a` must be defined)
```
Comment (robertology) [inline README.md]: `! empty()`

**Comment by adrian-enspired** [comment]
fixed.

**Pull request merged**